.packit.yaml

```diff
@@ -0,0 +1,48 @@
+---
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+specfile_path: mrack.spec
+
+# add or remove files that should be synced
+files_to_sync:
+  - mrack.spec
+  - README.md
+  - .packit.yaml
+
+# name in upstream package repository/registry (e.g. in PyPI)
+upstream_package_name: mrack
+# downstream (Fedora) RPM package name
+downstream_package_name: mrack
+
+actions:
+  create-archive:
+    - "python3 setup.py sdist --dist-dir ."
+    - "sh -c 'echo mrack-$(python3 setup.py --version).tar.gz'"
+  get-current-version:
+    - "python3 setup.py --version"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets: [fedora-stable]
```

Comment:
I suggest using fedora-all here since you are going to update your package in rawhide and so that you're not surprised about passing builds on stable Fedora and suddenly build failures in rawhide after a release.

Reply:
Thanks will update